marklogic-data-hub-central/ui/src/pages/TilesView.tsx

```diff
@@ -83,8 +87,6 @@ const TilesView = (props) => {
             setView(views[id])
     }
 
-    const location: any = useLocation();
-
     useEffect(() => {
```

Comment:
@wooldridge, the default view for explorer should be zerostate page, right now if you click on explorer icon from the toolbar it is being redirected to overview page. I did some debugging and adding this code inside the useEffect in tilesview.tsx file is fixing the issue. Please push those changes and I will test them again, also let me know if you run into any issues regarding this.

```
if(props.id == 'explore')
                setView(views[props.id], true)
            else
                setView(views[props.id])
```

Reply:
@Sanjeevani19, thanks so much for checking this! Thanks, @briantang, too!